src/ansys/fluent/core/session.py

```diff
@@ -118,7 +118,7 @@ def build_from_fluent_connection(
     ):
         """Build a BaseSession object from fluent_connection object."""
         self.fluent_connection = fluent_connection
-        self._remote_file_handler = remote_file_handler
+        self.remote_file_handler = remote_file_handler
```

Comment:
We were going to reinstate `upload` and `download` in this _interface_. We were going to monkey patch the interface depending on whether the functionality is available.

Reply:
Yes, I will refer existing example of `__dir__` method.